src/Spiffy.Monitoring/EventContext.cs

```diff
@@ -331,7 +331,10 @@ private void GenerateKeysIfNecessary(Dictionary<string, string> keyValuePairs)
 
         private static string GetKeyValuePairsAsDelimitedString(Dictionary<string, string> keyValuePairs)
         {
-            return string.Join(" ", keyValuePairs.Select(kvp =>
+            return string.Join(" ", keyValuePairs
+                // Values with length over DeprioritizedValueLength will come later (true > false).
+                .OrderBy(pair => pair.Value.Length > Configuration.DeprioritizedValueLength)
```

Comment:
it might be more obvious to use integral values instead:
`value.Length <= Configuration.DeprioritizedValueLength ? 0 : 1`

Reply:
Done in new commit